hack/make.sh

```diff
@@ -9,9 +9,13 @@ DIR="$( cd "$( dirname "$0" )/.." && pwd )"
 cd "$DIR/"
 
 CONTAINERD_VERSION=
+REQUIRED_CONTAINERD_VERSION="1.0.3"
```

Comment:
I am wondering if we could directly use `CONTAINERD_VERSION ` rather than `REQUIRED_CONTAINERD_VERSION `?

Or add some annotation for both two variables and tell code readers the difference between them.

Reply:
`CONTAINERD_VERSION` is the version of containerd installed in this machine, getting from command `containerd -v ...`. If `CONTAINERD_VERSION` not equals to required version, this script will install the required version.
Maybe we can change `CONTAINERD_VERSION` to `CURRENT_CONTAINERD_VERSION` for better semantics? @allencloud

Reply:
From your explanation, I have got what you mean. While I think it is OK for the variables, but there is still some comment required in the script. WDYT?

Reply:
You are right. First reading these code may be confused, I've added a comment in next commit. @allencloud